CMakeLists.txt

```diff
@@ -127,6 +127,13 @@ else()
     message(STATUS "UFE not found.")
 endif()
 
+find_package(MayaUsd OPTIONAL_COMPONENTS)
```

Comment:
I don't think you need the `OPTIONAL_COMPONENTS` - that is used when you want to specific a list of components which are optional. Since you didn't add `REQUIRED` this package is optional.